versions/ufo3/lib.plist.md

```diff
@@ -58,6 +58,12 @@ The dictionary may contain glyph names that are not in the font. The dictionary
 </dict>
 ```
 
+#### public.openTypePostUnderlinePosition
+
+This defines an integer position for underline, defined as the top of the underline from the baseline (the [OpenType post table definition]). If present, this value should be used for `underlinePosition` of the OpenType `post` and `CFF2`. If `postscriptUnderlinePosition` is not defined, this value should be used to calculate the `Type 1` and OpenType `CFF` `underlinePosition`; if `postscriptUnderlinePosition` is present, that value must be used for `Type 1` and `CFF`. 
```

Comment:
> and `CFF2`

I believe CFF2 does not contain any UnderlinePosition field (only CFF 1.0 has that), it will implicitly use the existing value from the `post` table. So I think you don't need to mention CFF2 here.

Reply:
> if `postscriptUnderlinePosition` is present, ...

I'd make more explicit that the following logic applies only when _both_ fields are present, e.g.

> if `postscriptUnderlinePosition` is also present in addition to `public.openTypePostUnderlinePosition`, ...

Reply:
Right, thank you.